.github/workflows/fill_station_action.yml

```diff
@@ -0,0 +1,119 @@
+name: Create and publish the Fill Station Docker image
+
+# Configures this workflow to run every time a change is pushed to the branch called `main` or a pull request to `main` is created
+on:
+  push:
+    branches: ['main']
+  pull_request:
+
+# Defines two custom environment variables for the workflow. These are used for the Container registry domain, and a name for the Docker image that this workflow builds.
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository_owner }}/fill-station
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        arch: [amd64, arm64]
+    steps:
+      - name: Prepare
+        run: |
+          platform=linux/${{ matrix.arch }}
+          echo "PLATFORM_PAIR=${platform//\//-}" >> $GITHUB_ENV
+
+      # Checks-out your repository under $GITHUB_WORKSPACE, which is the CWD for
+      # the rest of the steps
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - uses: bazel-contrib/setup-bazel@0.8.5
+        with:
+          # Avoid downloading Bazel every time.
+          bazelisk-cache: true
+          # Store build cache per workflow.
+          disk-cache: ${{ github.workflow }}
+          # Share repository cache between workflows.
+          repository-cache: true
+
+        # test for amd64 (can't test for arm64)
+      - name: Build and Test with Bazel
+        run: |
+          if [[ "${{ matrix.arch }}" == "amd64" ]]; then
+            bazel test //fill:all
+          elif [[ "${{ matrix.arch }}" == "arm64" ]]; then
+            sudo apt-get install g++-11-arm-linux-gnueabihf
+            sudo apt install gcc-11-arm-linux-gnueabihf
+            bazel build //fill:all --platforms=platform:rpi4 --cpu="aarch64"
+          fi
+          sudo mkdir -p ./fill/output
+          sudo cp bazel-bin/fill/fill_station ./fill/output
```

Comment:
Do you need these two lines at all? Can't you delete them and just change the Dockerfile to copy from bazel-bin/fill/fill_station instead of ./fill/output?

Reply:
Docker can't follow symlinks (which bazel-bin is), so it's easier to move it to a more convenient directory